README.md

```diff
@@ -4,13 +4,22 @@
 # skalarprodukt [![Status Umbra][status-umbra]][andivionian-status-classifier]
 Experimental multidimensional arrays for F#
 
+## Dependencies
+
+This project uses ![Paket](https://fsprojects.github.io/Paket/) dependency manager. Before opening the solution or building the project, you should install Paket bootstrapper into .paket directory and download the dependencies using the following commands:
```

Comment:
Also, we're following 80-char width in this Markdown document; please follow the rules.